### ISSUE TITLE
_user_id is never defined when initiating Client

### ISSUE THREAD

**Issue by jbcirs** [shauntarves/wyze-sdk#181]
`self._user_id` is never defined to a value when initiating `Client`. `create_access_code` will fail unless you manually define it. Current workaround is define client.locks._user_id = "your_user_guid"

**Comment by shauntarves** [comment]
Yes, thank you for pointing this out. This changed when the authentication schema changed from the user/pass to the API token. The user id simply isn't returning from the service during this new flow.

I'll make a note to update the docs
